src/profileSettings.js

```diff
@@ -50,6 +50,14 @@ class ProfileSettings extends MagisterThing {
 		original = original || this._magister._options.password
 		const schoolUrl = this._magister.school.url
 
+		if (!original) {
```

Comment:
I don't really think these checks are necessary, and if we actually do want them I guess we should do them everywhere in the code. But I kinda feel like the documentation tells users enough and apart from the we should just stick to the [GIGO principle](https://en.wikipedia.org/wiki/Garbage_in,_garbage_out).

But this is open for discussion :P

Reply:
Yeah true, I kind of did this to debug the function because the output on error was a bit strange. But still, now that it works it's imo better to stick to something that is already implemented then to write a BUNCH of extra code at this stage.